### ISSUE TITLE
Cannot find module '1password'

### ISSUE THREAD

**Issue by lpgeiger** [stayradiated/onepasswordjs#3]
Testing with the example code in documentation. Using node v8.1. Other packages import with no issues.
What would you suggest trying? 
````
module.js:487
    throw err;
    ^

Error: Cannot find module '1password'
    at Function.Module._resolveFilename (module.js:485:15)
    at Function.Module._load (module.js:437:25)
    at Module.require (module.js:513:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/Users/LUCAS/code/test/pass.js:1:74)
    at Module._compile (module.js:569:30)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:503:32)
    at tryModuleLoad (module.js:466:12)
    at Function.Module._load (module.js:458:3)
````

**Comment by stayradiated** [comment]
I haven't tried this module with node 8 yet. I'll test it out and see if there is anything I need to change.

**Comment by stayradiated** [comment]
I had accidentally broken the `main` property in the package.json. 

It's fixed in `v0.4.0`.

Thank you for bringing this to my attention :)
